Ivan branch

xaaniyaro opened the pull request:
- Movido el turno del jugador al lado izquierdo
- Arreglado el detalle del botón de logout
- Movido el botón de continuar a una zona constante
- Cambiado el texto del botón de listo
- Movido a la escena "Top10" el aviso del mejor turno
- Movido el botón del avatar a un lado del avatar de bienvenida
- Centrada burbuja de texto con la pantalla

xaaniyaro committed:
UI fixes and mission fix

- Se corrigió la posición del botón "Continuar" en todas las escenas.
Se movieron los elementos de la pantalla de "Menu".
- Se arreglaron las misiones 5 y 6

xaaniyaro committed:
Top3 movido de lugar

Se movió el aviso de mejor turno a la pantalla de la Tabla de Clasificación

xaaniyaro committed:
Botón "Listo" cambiado

Texto de botón "Listo" cambiado a "Revisar"

napo-lazo commented:
Todo se ve muy bien, lo único seria mover el botón de continuar al mismo lugar que el de revisar para tener consistencia a través de las preguntas.

xaaniyaro committed:
continueButton fix

Se cambió el prefab del botón para que fuera consistente con la posición de todas las escenas, y se movió hacia la parte inferior derecha como en las escenas de drag'n'drop
